src/main/java/org/craftercms/studio/impl/v1/util/ContentUtils.java

```diff
@@ -45,7 +45,7 @@ public static void release(InputStream in) {
                 in.close();
             }
         } catch (IOException e) {
-            logger.error("Failed to release a resource.", e);
```

Comment:
Any reason why not consolidating these 3 methods into a single one taking a Closeable param?

Reply:
Fixed.